apps/preview/nuxt/pages/showcases/Banners/Hero.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    :style="variables"
+    class="md:flex md:flex-row-reverse min-h-[600px] max-w-[1536px] mx-auto bg-[url:var(--sf-img-mobile)] md:bg-[url:var(--sf-img-desktop)] bg-center bg-cover bg-no-repeat"
+  >
+    <div class="flex flex-col md:basis-2/4 md:items-stretch md:overflow-hidden">
+      <img :src="headphones" alt="Headphones" class="h-full object-cover object-left" />
+    </div>
+    <div class="p-4 md:p-10 md:flex md:flex-col md:justify-center md:items-start md:basis-2/4">
+      <p class="typography-text-xs md:typography-text-sm font-bold tracking-widest text-neutral-500">Feel the music</p>
+      <h2 class="typography-headline-2 md:typography-headline-1 md:leading-[67.5px] font-bold mt-2 mb-4">
+        New Wireless Pro
+      </h2>
+      <p class="typography-text-base md:typography-text-lg">
+        Spatial audio. Adjustable ear cups. On-device controls. All-day battery.
+      </p>
+      <div class="flex flex-col md:flex-row gap-4 mt-6">
+        <SfButton> Order now </SfButton>
+        <SfButton variant="secondary"> Show more </SfButton>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { SfButton } from '@storefront-ui/vue';
+import headphones from '@assets/hero-headphones.png';
+import mobileBackground from '@assets/hero-bg-mobile.png';
+import desktopBackground from '@assets/hero-bg.png';
+
+const variables = {
+  '--sf-img-mobile': `url(${mobileBackground})`,
+  '--sf-img-desktop': `url(${desktopBackground})`,
+};
```

Comment:
I'm wondering if we could use [`<picture>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) element with two [sources](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source) for each screen size. Then instead of having those as a background image they would be two sources of one `<picture>` with proper `media` attributes. Do you think it would worth to use it or we just want to keep it as a bg?

Reply:
I can try it and check it will be the same :)

Reply:
This `<picture>` solution looks pretty neat to me. However, I'm wondering if we could somehow connect source media with Tailwind breakpoint values. But this could be further discussed in the future.